### ISSUE TITLE
[BUG] "cms check" error with django-constance package

### ISSUE THREAD

**Issue by ton77v** [django-cms/django-cms#7412]
## Description

"cms check" fails if there's django-constance package installed

## Steps to reproduce

1. Install django-cms==3.11 and Django==4.0.4 and django-constance==2.9.1
2. python manage.py cms check

## Actual behaviour

PlaceholderField
================
Checker failed, see traceback [ERROR]
...     ph_fields = [field for field in model._meta.get_fields() if isinstance(field, PlaceholderField)]
AttributeError: 'Meta' object has no attribute 'get_fields'

## Do you want to help fix this issue?
Issue can be solved adding a line in "check.py":
```
for model, model_admin in site._registry.items():
    if getattr(model._meta, 'get_fields', None):
        ph_fields = [field for field in model._meta.get_fields() if isinstance(field, PlaceholderField)]

```

**Comment by fsbraun** [comment]
@ton77v Thanks for reporting this! 

As an open source project, django CMS is only as strong as its community. Can I encourage you to crate a pull request for this fix? 

Also, please join our slack channel [#workgroup-pr-review](https://django-cmsworkspace.slack.com/archives/C01J7H87KFC). You'll find many people there to support you. Once you've created the PR, you'll find people to review it and get merged into the development branch of django CMS.

**Comment by fsbraun** [comment]
Fixed with #7413 . Thank you very much for this contribution, @ton77v !
